src/index.js

```diff
@@ -23,13 +23,25 @@ export default function maybeExtendPromise(Promise) {
   // handled Promises to their corresponding fulfilledHandler.
   let forwardingHandler;
   function handler(p, operation, ...args) {
```

Comment:
Should this be `handle` instead of `handler`? From `handler` I would guess that it obtains or creates the needed handler and then returns it. I think we need a verb.

Reply:
Done.